cave_api/cave_api/examples/kitchen_sink.py

```diff
@@ -408,6 +408,31 @@ def execute_command(session_data, socket, command="init", **kwargs):
                             "enabled": True,
                             "help": "Select all relevant items",
                         },
+                        "listToggleExample": {
+                            "name": "List Toggle Example",
+                            "type": "selector",
+                            "variant": "listToggle",
+                            "options": {
+                                "field_a": {"name": "Field A"},
+                                "field_b": {"name": "Field B"},
+                            },
+                            "help": "Toggle all relevant items",
+                        },
+                        "listCheckboxExample": {
+                            "name": "List Checkbox Example",
+                            "type": "selector",
+                            "variant": "listCheckbox",
+                            "options": {
+                                "option_a": {"name": "Option A", "field": "Field A"},
+                                "option_b": {"name": "Option B", "field": "Field A"},
+                                "option_c": {"name": "Option C", "field": "Field A"},
+                                "option_d": {"name": "Option D", "field": "Field B"},
+                                "option_e": {"name": "Option E", "field": "Field B"},
+                                "option_f": {"name": "Option F", "field": "Field B"},
```

Comment:
@connor-makowski, @aiizhao: I'm thinking we might want a `field` key that references the `field`'s `name`, but that might require a new `selector` special property. For example:
```py
{
    // ...
    "options": {
        "option_a": {"name": "Option A", "field": "field_a"},
        "option_b": {"name": "Option B", "field": "field_a"},
        "option_c": {"name": "Option C", "field": "field_a"},
        "option_d": {"name": "Option D", "field": "field_b"},
        "option_e": {"name": "Option E", "field": "field_b"},
        "option_f": {"name": "Option F", "field": "field_b"},
    },
    "fields": {
        "field_a": {"name": "Field A"},
        "field_b": {"name": "Field B"},
    },
}
```

Reply:
I am torn here. I think we can reduce size with your solution @luisvasq ,but at the cost of some complexity. I am not sure how I would want to balance that trade off with this particular situation. Once we commit either way, we are locked in until at least 3.0

@mrkwanzaa any thoughts from your sidd?

Reply:
Makes sense. My initial thought was to use a key or ID as a reference instead of the attribute name itself, but I know this adds some complexity with the need for an extra key-value pair.